Cargo.toml

```diff
@@ -25,19 +29,25 @@ tokio = { version = "1.37.0", features = ["full"] }
 esplora-client = { git = "https://github.com/BitVM/rust-esplora-client" }
 serde_json = "1.0.116"
 lazy_static = "1.4.0"
-bitcoin-script-stack = { git = "https://github.com/FairgateLabs/rust-bitcoin-script-stack"}
+bitcoin-script-stack = { git = "https://github.com/FairgateLabs/rust-bitcoin-script-stack" }
 rand = "0.8.5"
 rand_chacha = "0.3.1"
 once_cell = "1.19.0"
 dotenv = "0.15.0"
 aws-sdk-s3 = "1.40.0"
 regex = "1.10.5"
 blake3 = "=1.5.1"
+paste = "1.0.15"
```

Comment:
I think all the changes to the `Cargo.toml` that are not autoformatting (due to Taplo) we can drop.

Hence keep only this.